### ISSUE TITLE
Add more than one file

### ISSUE THREAD

**Issue by nitrocode** [suzuki-shunsuke/github-action-renovate-config-validator#699]
It would be nice to be able to add more than a single file for cases of a centralized config

I'm doing this for now with a matrix

```yaml
jobs:
  config-files:
    runs-on: ubuntu-latest
    outputs:
      files: ${{ steps.file_list.outputs.files }}

    steps:
      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4

      - name: Find JSON and JSON5 files
        id: file_list
        run: |
          set -x
          files=$(find . -type f \( -name "*.json" -o -name "*.json5" \) | sed 's|^\./||')
          files_json=$(echo "$files" | jq -R -s -c 'split("\n")[:-1]')
          echo "files=$files_json" >> $GITHUB_OUTPUT

  validate:
    needs: config-files
    runs-on: ubuntu-latest
    strategy:
      matrix:
        file: ${{ fromJson(needs.config-files.outputs.files) }}
    steps:
      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
      - uses: suzuki-shunsuke/github-action-renovate-config-validator@b54483862375f51910a60c4f498e927d4f3df466 # v1.0.1
        with:
          config_file_path: ${{ matrix.file }}
```

If supporting more than one file is not considered, please consider documenting the above approach

One improvement would be to only run files when they are modified.

**Comment by suzuki-shunsuke** [comment]
Thank you for your feedback.

How about this?

- https://github.com/suzuki-shunsuke/github-action-renovate-config-validator/pull/701

You can try `suzuki-shunsuke/github-action-renovate-config-validator@feat-multiple-files`.

**Comment by suzuki-shunsuke** [comment]
Hmm. It's a bit difficult to implement glob including double star using shell script.
Maybe we should rewrite this action as JavaScript action.

**Comment by nitrocode** [comment]
Ok no worries. This is still very good. I can always use a different action to pick up all the files and then pass them into this action. Thank you.

**Comment by suzuki-shunsuke** [comment]
Thank you.
I've merged #701 and released v1.1.0.
https://github.com/suzuki-shunsuke/github-action-renovate-config-validator/releases/tag/v1.1.0
